ui/templates/provision.j2

```diff
@@ -160,6 +160,14 @@
                               title="Joyent Minimal Native Zone">
                               joyent-minimal
                             </option>
+                            <!-- TODO: Delete this line to enable provisioning builder zones
+                            <option
```

Comment:
There's a tiny part of me that wonders if this can somehow detect if the PI it's running has builder zone support, but I can't think of a good way off the top of my head.